conf-example.json

```diff
@@ -1,15 +1,26 @@
 {
   "host": "127.0.0.1",
   "port": 8080,
-  
+
   "uploadsDirectory": "uploads/",
   "workingDirectory": "./",
-  
+
   "database" : {
     "host" : "localhost",
     "port" : 5432,
     "username" : "postgres",
     "password" : "secret",
     "database" : "tableaux"
+  },
+
+  "auth": {
+    "realm": "keycloak-realm",
+    "bearer-only": true,
+    "auth-server-url": "http://localhost:9999/auth",
+    "ssl-required": "external",
+    "resource": "grud-backend",
```

Comment:
In KeycloakAuthHandler.scala there is a check for a config value "issuer". This config entry is missing in the example config.